### PR TITLE
More natural phrasing in README.rst

### PR DESCRIPTION
**Pull request by cedricrommel**
Sorry, I did not see this when the PR was opened, so just creating a new PR 😬 

These are just small changes to the new README.rst in places where I found the phrasing a bit unnatural. Also, I removed the part mentioning the decoding of MEG, SEG and ECoG, since the lib does not support this other data modalities yet AFAIK.

### CODE REVIEW

**Review by bruAristimunha — CHANGES_REQUESTED** [review]
Your changes look great to me! But, you may need to change also the [index.rst](https://github.com/braindecode/braindecode/blob/master/docs/index.rst) to affect the new webpage.

File: README.rst [hunk 1]
```diff
@@ -19,11 +19,12 @@
 Braindecode
 ===========
 
-`Braindecode`_ is an open-source Python toolbox for decode neurophysiological data such as MEG, EEG, sEEG,
-ECoG, and more with Deep Learning models. It includes modules for data input/output, preprocessing,
-visualization, data augmentation, classification, regression, sampler and much more!
+`Braindecode`_ is an open-source Python toolbox for decoding raw EEG data with
```
Comment (sliwy) [inline README.rst:22]: I would keep ECoG as we have two examples analyzing ECoG data, for example this one: [plot_bcic_iv_4_ecog_cropped.py](https://github.com/braindecode/braindecode/blob/master/examples/plot_bcic_iv_4_ecog_cropped.py)

**Review by bruAristimunha — APPROVED** [review]

**Comment by bruAristimunha** [comment]
I made a small commit to change this ECoG and MEG issue (because of the MNE), and to copy the information to index.rst for the new documentation. @sliwy or @cedricrommel, for me you can apply the merge in PR.

File: README.rst (hunk 1 above)
Comment (cedricrommel) [reply README.rst]: Sure! Didn't know about those 😅  Cool stuff!

**Comment by cedricrommel** [comment]
Sorry guys, I am a bit taken by the NeurIPS rebuttal phase. I would recommending not merging this PR yet as I found some other copy-paste mistakes in the FAQ if I remember well. Would be nice to gather all these small fixes in this PR. Will be back actively next week!

**Comment by robintibor** [comment]
I'm not so sure about the changes removing EEG/MEG etc, like brain signals can also be fMRI, which we don't really do at all so far?

File: README.rst [hunk 2]
```diff
@@ -19,12 +19,13 @@
...
-ECoG, and more with Deep Learning models. It includes modules for data input/output, preprocessing,
-visualization, data augmentation, classification, regression, sampler and much more!
+Braindecode is an open-source Python toolbox for decoding raw neurophysiological
+data with deep learning models. It includes dataset fetchers, data preprocessing and
+visualization tools, as well as implementations of several deep learning
+architecture and data augmentations for EEG, ECoG, MEG and others analysis.
 
-This toolbox focus only in decode raw time-domain EEG. For EEG researchers who want to work with deep learning and
-deep learning researchers who want to work with EEG data.
+For brain signal researchers who want to work with deep learning and
```
Comment (agramfort) [inline README.rst:27]:
```suggestion
For neuroscientists who want to work with deep learning and
```

File: docs/index.rst [hunk 3]
```diff
@@ -24,13 +24,13 @@ Braindecode Homepage
...
-    convolutional architecture, recurrent architecture, attention architecture.
+   Braindecode is an open-source Python toolbox for decoding raw neurophysiological
+   data with deep learning models. It includes dataset fetchers, data preprocessing and
+   visualization tools, as well as implementations of several deep learning
+   architecture and data augmentations for EEG, ECoG, MEG and others analysis.
 
-    It includes modules for data input/output, preprocessing, visualization,
-    data augmentation, classification, regression, sampler and
-    much more!
+   For brain signal researchers who want to work with deep learning and
```
Comment (agramfort) [inline docs/index.rst:32]:
```suggestion
   For neuroscientists who want to work with deep learning and
```

File: README.rst [hunk 4]
```diff
@@ -19,12 +19,13 @@
 Braindecode
 ===========
 
-`Braindecode`_ is an open-source Python toolbox for decode neurophysiological data such as MEG, EEG, sEEG,
-ECoG, and more with Deep Learning models. It includes modules for data input/output, preprocessing,
-visualization, data augmentation, classification, regression, sampler and much more!
+Braindecode is an open-source Python toolbox for decoding raw neurophysiological
+data with deep learning models. It includes dataset fetchers, data preprocessing and
+visualization tools, as well as implementations of several deep learning
+architecture and data augmentations for EEG, ECoG, MEG and others analysis.
```
Comment (robintibor) [inline README.rst:25]:
```suggestion
Braindecode is an open-source Python toolbox for decoding raw electrophysiological brain
data with deep learning models. It includes dataset fetchers, data preprocessing and
visualization tools, as well as implementations of several deep learning
architectures and data augmentations for analysis of EEG, ECoG and MEG.
```

File: docs/index.rst [hunk 5]
```diff
@@ -24,13 +24,13 @@ Braindecode Homepage
 
 .. rst-class:: h4 text-center font-weight-light my-4
 
-    Braindecode is an open-source toolbox for decode raw neurophysiological
-    data such as MEG, EEG, sEEG, ECoG with Deep Learning models, such as
-    convolutional architecture, recurrent architecture, attention architecture.
+   Braindecode is an open-source Python toolbox for decoding raw neurophysiological
+   data with deep learning models. It includes dataset fetchers, data preprocessing and
+   visualization tools, as well as implementations of several deep learning
+   architecture and data augmentations for EEG, ECoG, MEG and others analysis.
```
Comment (robintibor) [inline docs/index.rst:30]:
```suggestion
Braindecode is an open-source Python toolbox for decoding raw electrophysiological brain
data with deep learning models. It includes dataset fetchers, data preprocessing and
visualization tools, as well as implementations of several deep learning
architectures and data augmentations for analysis of EEG, ECoG and MEG.
```

**Comment by cedricrommel** [comment]
Alright guys, I just fixed the few things I saw but did not have time to deal with last week :)
Thanks for helping to make the homepage text neat!! 🚀 

Good to go as far as it concerns me!

**Comment by robintibor** [comment]
yes looks good to me what do others think?

**Review by agramfort — APPROVED** [review]

**Comment by cedricrommel** [comment]
Merging it then!

**Pull request merged**